spec/checklists/locations

```diff
@@ -0,0 +1,63 @@
+1.GET Method
+Valid request:
+ 1.1 Verify GET/locations response include list of all locations
+ 1.2 Verify GET/locations status code is 201
+ 1.3 Verify GET/location/:id response include required information of location(id, name, description) in response
+ 1.4 Verify GET/locations/:id status code is 201
+ 1.5 Verify GET/location/:name response include required information of location(id, name, description)
+ 1.6 Verify GET/locations/:name status code is 201
+
+ Invalid request:
+ 1.7 Verify authorized user GET/locations/:id(non-exist id, float number, negative number, number and special characters/letters, location that was deleted, empty, null) status code is 404
+ 1.8 Verify authorized user GET/locations/:name(non-exist name, deleted/changed, empty, length less 2, length more 1000) status code is 404
+ 1.9 Verify non-authorized user GET/locations status code is 401
+ 1.10 Verify non-authorized user GET/locations/:id status code is 401
+ 1.11 Verify non-authorized user GET/location/:name status code is 401
+
+
+ 2.POST Method
+ Valid request:
+ 2.1 Verify POST/locations responce include all created data(name, description)
+ 2.2 Verify POST/locations status code is 201
+ 2.3 Verify POST/location length name 5 &  length description 1000 status code 201
```

Comment:
Verify POST/location with length of  name 5 and length of description 1000 status code 201